docker/venice-server/single-dc-configs/cluster.properties

```diff
@@ -0,0 +1,7 @@
+persistence.type=ROCKS_DB
```

Comment:
NIT: there are multiple cluster.properties files for controller and server with the same keys in configs, but arranged in different orders. can we rearrange them for better readability?

Reply:
This is on purpose. The goal is to keep building docker images simple.  We can evolve them in the future if required.